OpenKeychain/src/main/java/org/sufficientlysecure/keychain/ui/PassphraseDialogActivity.java

```diff
@@ -107,15 +107,22 @@ protected void onCreate(Bundle savedInstanceState) {
 
         // this activity itself has no content view (see manifest)
         RequiredInputParcel requiredInput = getIntent().getParcelableExtra(EXTRA_REQUIRED_INPUT);
-        if (requiredInput.mType != RequiredInputType.PASSPHRASE) {
+        if (requiredInput.mType != RequiredInputType.BACKUP_CODE &&
+            requiredInput.mType != RequiredInputType.NUMERIC_9X4 &&
+            requiredInput.mType != RequiredInputType.NUMERIC_9X4_AUTOCRYPT &&
+            requiredInput.mType != RequiredInputType.PASSPHRASE &&
+            requiredInput.mType != RequiredInputType.PASSPHRASE_AUTH &&
+            requiredInput.mType != RequiredInputType.PASSPHRASE_SYMMETRIC) {
```

Comment:
The thing that would help me when reading this would be to use (!(A || B)) instead.